google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractReadContext.java

```diff
@@ -790,7 +790,7 @@ ResultSet readInternalWithOptions(
             SpannerRpc.StreamingCall call =
                 rpc.read(builder.build(), stream.consumer(), session.getOptions());
             call.request(prefetchChunks);
-            stream.setCall(call, selector != null && selector.hasBegin());
+            stream.setCall(call, /* withBeginTransaction = */ builder.getTransaction().hasBegin());
```

Comment:
Is this typo? Strange way to write an inline comment.

Reply:
No, it's not a typo. It's the closest we get to named parameters in Java. We sometimes use it for (boolean) arguments that otherwise are not obvious what they are for.

Reply:
Thank you for clarification